demos/gesture_recognition_demo/python/gesture_recognition_demo/person_detector.py

```diff
@@ -78,19 +80,8 @@ def wait_request(self, req_id):
         if result is None:
             return None
 
-        initial_h, initial_w = self.last_sizes
         scale_h, scale_w = self.last_scales
 
-        out = self._process_output(result, initial_h, initial_w, scale_h, scale_w)
-
-        return out
-
-
-    def __call__(self, frame):
-        """Runs model on the specified input"""
-
-        in_frame, initial_h, initial_w, scale_h, scale_w = self._prepare_frame(frame)
-        result = self.infer(in_frame)
-        out = self._process_output(result, initial_h, initial_w, scale_h, scale_w)
+        out = self._process_output(result)
 
         return out
```

Comment:
You have `scale_h, scale_w = self.last_scales` left. While you are here, this whole part should be replaced with just `return self._process_output(result)`